internal/cmd/shell.go

```diff
@@ -71,7 +71,7 @@ func shell(_ *cobra.Command, cmdArgs []string) error {
 		if exitErr, ok := err.(wsep.ExitError); ok {
 			os.Exit(exitErr.Code)
 		}
-		return xerrors.Errorf("run command: %w", err)
```

Comment:
Ideally we would be able to handle this better. This failure case _should_ propagate an error such that we exit with some nonzero code. I think there may be a way for us to hook into the cobra error handling so we can fix the ugly error formatting.